tests/compile/srcjar/expected

```diff
@@ -1,4 +1,4 @@
 -rw----     2.0 fat        0 bx  0% stor 20100101.000000 META-INF/
--rw----     2.0 fat       48 b-  0% stor 20100101.000000 META-INF/MANIFEST.MF
--rw----     2.0 fat      389 b-  0% stor 20100101.000002 Example$.class
 -rw----     2.0 fat      328 b-  0% stor 20100101.000002 Example.class
+-rw----     2.0 fat      389 b-  0% stor 20100101.000002 Example$.class
+-rw----     2.0 fat       48 b-  0% stor 20100101.000000 META-INF/MANIFEST.MF
```

Comment:
Since you're sorting the expected files now, you shouldn't need to change them at all.